packages/flutter/lib/src/painting/box_border.dart

```diff
@@ -314,6 +314,16 @@ class Border extends BoxBorder {
        assert(bottom != null),
        assert(left != null);
 
+  /// Creates a border whose sides are all the same.
+  ///
+  /// The `side` argument must not be null.
+  const Border.uniformSide(BorderSide side)
```

Comment:
At first glance, I would prefer something shorter like `Border.uniform()`. Its implied to be a uniform side, because the only parameters we ever pass to border are the side, right?

Reply:
Yep, makes sense.  Done.